drudapi/docker.go

```
@@ -0,0 +1,20 @@
+package drudapi
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+// DockerCompose serves as a wrapper to docker-compose
+func DockerCompose(arg ...string) {
```

this should return the error so we can handle it if we need to